public/app/features/dashboard/dashboard_model.ts

```diff
@@ -272,24 +272,41 @@ export class DashboardModel {
       let panel = this.panels[i];
       if (panel.repeat) {
         this.repeatPanel(panel, i);
+      } else if (panel.panels) {
+        // find repeate panel on collapsed state
+        let repeatPanel;
+        for (let j = 0; j < panel.panels.length; j++) {
+          if (panel.panels[j].repeat) {
+            repeatPanel = panel.panels[j];
```

Comment:
this also looks strange, why just a repeat a single panel in the collapsed row?

Reply:
I agree. There can be several such panels in a collapsed row. This is my omission - I'll fix it.